src/hotspot/cpu/ppc/macroAssembler_ppc.hpp

```diff
@@ -161,6 +161,10 @@ class MacroAssembler: public Assembler {
   //
   // branch, jump
   //
+  // set dst to -1, 0, +1
```

Comment:
Added this comment:
```
// set dst to -1, 0, +1 as follows: if CCR0bi is "greater than", dst is set to 1,
// if CCR0bi is "equal", dst is set to 0, otherwise it's set to -1.
```